src/test/shared/credentials/accountId.test.ts

```diff
@@ -13,7 +13,7 @@ import { ext } from '../../../shared/extensionGlobals'
 describe('getAccountId', function () {
     let sandbox: sinon.SinonSandbox
 
-    const credentials: AWS.Credentials = ({} as any) as AWS.Credentials
+    const credentials: AWS.Credentials = {} as any as AWS.Credentials
```

Comment:
This file is being automatically modified by precommit

Reply:
Yeah prettier will alternate adding and removing parentheses for some reason.